src/software/SfM/main_openMVG2CMPMVS.cpp

```diff
@@ -77,7 +77,11 @@ bool exportToCMPMVSFormat(
 
       const Pose3 pose = sfm_data.GetPoseOrDie(view);
       Intrinsics::const_iterator iterIntrinsic = sfm_data.GetIntrinsics().find(view->id_intrinsic);
-
+      const IntrinsicBase * cam = iterIntrinsic->second.get();
+      if(cam->w() == 0 || cam->h() == 0)
```

Comment:
Yes, there is already a test for that before: `if (!sfm_data.IsPoseAndIntrinsicDefined(view))`.
Here, it's specific to the implementation of the Alembic file which doesn't have the information about valid or invalid intrinsics.
I need to add this notion in the Alembic exporter, but it was an easy fix without drawback (and which works with the reconstructions already computed).